Controller/MainMenuController.py

```diff
@@ -0,0 +1,52 @@
+# <========== Imports ==========>
+
+from __future__ import annotations
+from tkinter import Frame, PhotoImage, Tk
+
+# <========== Local Imports ==========>
+
+# Model Imports
+from Model.Question import Question
+
+# View Imports
+from View.WelcomePage import WelcomePage
+
+# Controller Imports
+from Controller.QuestionController import QuestionController
+
+# <========== Class ==========>
+
+class MainMenuController:
+
+    def __init__(self: MainMenuController, questions: list[Question]) -> None:
+        """ The constructor for MainMenuController class.
+        Create a view for the main menu.
+
+        Args:
+            self (QuestionController): Self.
+            questions (list[Question]): All Questions to display when start button is pushed.
+        """
+        self.questions: list[Question] = questions
+
+        self.root: Tk = Tk()
+        self.root.title("Quiz Game")
+        self.root.iconphoto(False, PhotoImage(file="img/logo-favicon.png"))
```

Comment:
This should get the absolute path of the file, in case the program is launched from another location (such as during the tests...)